src/SprykerEco/Zed/Heidelpay/Business/Adapter/Mapper/RequestToHeidelpay.php

```diff
@@ -53,6 +53,12 @@ public function map(HeidelpayRequestTransfer $requestTransfer, Request $heidelpa
             $requestTransfer->getCustomerPurchase()->getSecret()
         );
 
+        $riskInformationTransfer = $requestTransfer->getRiskInformation();
+
+        $heidelpayRequest->getRiskInformation()->setCustomerSince($riskInformationTransfer->getCustomerSince());
+        $heidelpayRequest->getRiskInformation()->setCustomerGuestCheckout($riskInformationTransfer->getIsCustomerGuest());
```

Comment:
Shouldn't this be `setIsCustomerGuest`?

Reply:
This is comming from `Heidelpay` namespace:
https://github.com/heidelpay/php-payment-api/blob/master/lib/ParameterGroups/RiskInformationParameterGroup.php#L73